dashboards/ns-server-dashboard.json

```diff
@@ -247,6 +247,18 @@
           "_base": "target"
         }
       ]
+    },
+    {
+      "title": "check_time message delay",
+      "_base": "panel",
+      "_targets": [
+        {
+          "datasource": "{data-source-name}",
+          "expr": "histogram_quantile(0.999, irate(cm_timer_lag_seconds_bucket[5m]))",
```

Comment:
Do you really want the 3 9's percentile? Maybe 50th, 90th and 99th?

Reply:
I like that .999 clearly shows an outlier in the stat. I can then edit the query in grafana to get other percentiles.

Reply:
Best case this runs every second and on average we're going to collect this statistic something around every 30 seconds. Asking for the 99.9th percentile is basically asking for the max - which I think is fine. It's probably reasonable to ask for the 90th percentile as well - essentially what are the bottom 3 numbers.